frontend/src/admin/components/sidebar/dropdown/dropdownItem.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {Action} from "../../../models/action";
+
+interface DropDownProps {
```

Comment:
The interface here says `DropDown` but everywhere else it is referred to as `Dropdown`.

Reply:
This is now implemented as Dropdown ✅